src/clients/ETHClient.ts

```diff
@@ -126,6 +135,58 @@ export class ETHClient {
     return new BigNumber(allowance.toString())
   }
 
+  public async bridgeTokens(params: BridgeParams): Promise<EthersTransactionResponse> {
+    const { fromToken, toToken, amount, recoveryAddress, ethAddress, signer, gasPriceGwei, gasLimit } = params;
+
+    const networkConfig = this.getNetworkConfig();
+    const rpcProvider = this.getProvider();
+    const contractAddress = this.getLockProxyAddress();
+    
+    const fromTokenId = fromToken.id;
+    const fromTokenAddress = appendHexPrefix(fromToken.tokenAddress);
+    const toTokenDenom = toToken.denom;
+    const decimals = fromToken.decimals.toNumber();
+
+    const swthAddress = ethers.utils.hexlify(AddressUtils.SWTHAddress.getAddressBytes(recoveryAddress, CarbonSDK.Network.MainNet));
```

Comment:
should validate `recoveryAddress` format before using it

Reply:
and should not name it as `swthAddress`, it implies this is a carbon address, which it is not.